analyzers/tests/SonarAnalyzer.UnitTest/LiveVariableAnalysis/RoslynLiveVariableAnalysisTest.cs

```diff
@@ -864,6 +865,22 @@ public class Sample
             context.ValidateEntry(new Captured("intParameter"));
         }
 
+        [TestMethod]
+        public void PropertyWithoutVariables()
+        {
+            var code = @"
+public class Sample
+{
+    private int target;
+    public int Property => target = 42;
+}";
+            var (tree, model) = TestHelper.Compile(code, false, AnalyzerLanguage.CSharp);
+            var method = tree.GetRoot().DescendantNodes().First(x => x.RawKind == (int)SyntaxKind.SimpleAssignmentExpression);
+            var cfg = ControlFlowGraph.Create(method.Parent, model);
+            var lva = new RoslynLiveVariableAnalysis(cfg);
+            new Context(cfg, lva).ValidateAllEmpty();
```

Comment:
We don't need to repeat this, that's what the `Context` does for us:
```suggestion
new Context(code, AnalyzerLanguage.CSharp).ValidateAllEmpty;
```